README.md

```diff
@@ -1,14 +1,22 @@
 <a name="top"></a>
 #  Polyfill Service
+Polyfill Service sends down only the polyfills that your browsers need, because browsers deserve the bare minimum. 
 [![license](https://img.shields.io/github/license/mashape/apistatus.svg)](https://img.shields.io/github/license/mashape/apistatus.svg)
 [![Coverage Status](https://coveralls.io/repos/github/reiniergs/polyfill-service/badge.svg?branch=codeCoverage)](https://coveralls.io/github/reiniergs/polyfill-service?branch=codeCoverage)
 [![Build Status](https://travis-ci.org/reiniergs/polyfill-service.svg?branch=master)](https://travis-ci.org/reiniergs/polyfill-service)
+## Getting Started
+To get started, [clone and build](#running-locally) the service. Polyfill Service is available in two flavors, as a [Java API](/polyfill-service-api/README.md) or [RESTful API](/polyfill-service-rest/README.md). With the Java API, you can customize how you want to fetch polyfills and even narrow down which polyfills to fetch. The REST API provides endpoints for you to work with different URL variables and query parameters. Both APIs fulfill the following conditions:
+* Does not require browsers to download unneeded polyfills
+* Does not require browsers to make blocking HTTP requests
```

Comment:
This should be deleted, because the service actually needs an additional blocking HTTP request even if the user’s browser supports all the necessary features. Thanks for catching this, @scottmo!
Source: https://philipwalton.com/articles/loading-polyfills-only-when-needed/ (User Agent Passing section)